collagraph/__init__.py

```diff
@@ -408,9 +503,10 @@ def is_equivalent_event_handler(val, other, key):
                 continue
 
             event_type = key_to_event(name)
-            self.renderer.remove_event_listener(dom, event_type, val)
+            events_to_remove[event_type] = val
```

Comment:
just wondering if it's possible that there are multiple events to remove for a single event type?

Reply:
Right, I guess we need to replicate the same data structure: dict with sets as values.

Reply:
Actually: the question is: do we want to support the following?

```python
class Comp(Component):
    def call_a(self, event):
        pass

    def call_b(self, event):
        pass

    def render(self):
        return h("x", {"on_event": {self.call_a, self.call_b}})
```

I think that we don't want that (also in the case that we add JSX-like syntax to specify components), so I think the code is fine as it is.

Reply:
Good call!